resources/deployments/controller/controller_deployments.go

```diff
@@ -127,6 +128,36 @@ func (d *DeploymentsController) GetDeploymentStats(w rest.ResponseWriter, r *res
 	d.view.RenderSuccessGet(w, stats)
 }
 
+func (d *DeploymentsController) AbortDeployment(w rest.ResponseWriter, r *rest.Request) {
+	l := requestlog.GetRequestLogger(r.Env)
+
+	id := r.PathParam("id")
+
+	if !govalidator.IsUUIDv4(id) {
+		d.view.RenderError(w, r, ErrIDNotUUIDv4, http.StatusBadRequest, l)
+		return
+	}
+
+	// Check if deployment is finished
+	isDeploymentFinished, err := d.model.IsDeploymentFinished(id)
+	if err != nil {
+		d.view.RenderInternalError(w, r, err, l)
```

Comment:
Shouldn't we respond with 409 Conflict perhaps? Would require docs update as well.